### PR TITLE
Refactor unused code and enhance test suite coverage

### PR DESCRIPTION
**Pull request by masanori-satake**
This change addresses the user's request to clean up unused code and ensure the test suite is comprehensive and passing.

Key changes:
1. **Unused Code Removal**: Localized module-level variables in `projects/studio/js/studio.js` and `projects/category-editor/js/category-editor.js` that were only used during the initialization phase.
2. **Test Suite Optimization**:
   - Renamed `tests/i18n.test_node.js` to `tests/i18n.test.js` to include it in the Jest runner.
   - Removed duplicate test cases for `visualPadEnd` and redundant report generation tests in `tests/logic.test.js`.
3. **Coverage Improvement**:
   - Added test cases for `logic.js` covering `stripEmojis` edge cases, empty log reports, and manual stop marker logic.
   - Added test cases for `db.js` covering lite-mode initialization, old log cleanup, and multi-add edge cases.
   - Line coverage for `logic.js` is now 100%, and `db.js` is 96%.
4. **Verification**:
   - All 214 unit tests pass.
   - All Playwright E2E tests pass.
   - Frontend UI was manually verified via Playwright scripts to ensure no regressions in Studio or Category Editor.

Implementation follows the 'Pure & Long-life Design' principle by improving code encapsulation without adding dependencies.

---
*PR created automatically by Jules for task [13018319199284235167](https://jules.google.com/task/13018319199284235167) started by @masanori-satake*

### CODE REVIEW

File: projects/studio/js/studio.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ const elements = {
     toggleConsoleBtn: document.getElementById('toggle-console')
 };
 
-let editorMod, metricsMod, previewMod, ioMod;
+let metricsMod;
```
[the remark this answers is not in the document]
Comment (masanori-satake) [reply projects/studio/js/studio.js]: @jules
PRコメントの妥当性を検証してください。
妥当ではないと判断した際には、一切コードを修正しないでください。
妥当であると判断した際には、PRコメントを参考に改善を検討してください。

File: tests/logic.test.js [hunk 2]
```diff
@@ -460,6 +460,11 @@ describe('Logic Module', () => {
             expect(generateReport(logs, { ...defaultOptions, adjust: 'abc' })).toBe(expected);
         });
 
+        test('prepareReportItems returns empty array when no logs', () => {
```
[the remark this answers is not in the document]
Comment (masanori-satake) [reply tests/logic.test.js:463]: @jules
PRコメントの妥当性を検証してください。
妥当ではないと判断した際には、一切コードを修正しないでください。
妥当であると判断した際には、PRコメントを参考に改善を検討してください。
